packages/app/src/app/pages/index.tsx

```diff
@@ -220,7 +220,7 @@ const RoutesComponent: React.FC = () => {
             <Route path="/cli/login" component={CLI} />
             <Route path="/client/login" component={MobileAuth} />
             <Route path="/vscode/login" component={VSCodeAuth} />
-            <Route path="/auth/zeit" component={VercelSignIn} />
+            <Route path="/auth/vercel" component={VercelSignIn} />
```

Comment:
Should we add a redirect from the old route to `/vercel` too?

Reply:
We're doing server side redirects for these routes, so it will be automatically redirected. But yeah better safe than sorry. I can add one!